cloud-sql-proxy/README.MD

```diff
@@ -49,6 +49,6 @@ You can use this initialization action to create a Dataproc cluster using a shar
 
 ## Important notes
 * Hive stores the metadata of all tables in it's metastore. It stores the contents of (non-external) tables in a HCFS directory, which is by default on HDFS. If you want to perist your tables beyond the life of the cluster, set "hive.metastore.warehouse.dir" to a shared locations (such as the Cloud Storage directory in the example above). This directory get's baked into the Cloud SQL metastore, so it is recommended to set even if you intend to mostly use external tables. If you place this direcotry in Cloud Storage. You may want to use the shared NFS consistency cache as well.
-* This initalization action can be reconfigured to only install a proxy on the master, if you only wish to have a shared metastore. Set `ENABLE_PROXY_ON_WORKERS` to 0.
-* It can be also be reconfigured to istall proxies without changing the Hive metasore, if you only wish to have the proxies. Set `ENABLE_CLOUD_SQL_METASTORE` to 0 and do not set the `hive-metastore-instance` metadata key.
+* This initalization action can be reconfigured to only install a proxy on the master, if you only wish to have a shared metastore. Set the `enable-cloud-sql-proxy-on-workers` metadata key to `0`.
+* It can be also be reconfigured to istall proxies without changing the Hive metasore, if you only wish to have the proxies. Set the `enable-cloud-sql-hive-metastore` metadata key to `0` and do not set the `hive-metastore-instance` metadata key.
```

Comment:
Since you're touching this line anyway, "istall" -> "install"

Reply:
Sure, I'll update the PR with the typo fix